src/eval_llvm.cpp

```diff
@@ -612,14 +612,18 @@ void jitc_llvm_ray_trace(uint32_t func, uint32_t scene, int occluded,
         valid = steal(jitc_var_new_op(JitOp::Select, 3, deps_2));
     }
 
+    bool within_vcall = thread_state(JitBackend::LLVM)->recording_vcall;
+
     jitc_log(InfoSym, "jitc_llvm_ray_trace(): tracing %u %sray%s%s%s", size,
              occluded ? "shadow " : "", size != 1 ? "s" : "",
              placeholder ? " (part of a recorded computation)" : "",
              double_precision ? " (double precision)" : "");
 
     Ref op = steal(jitc_var_new_stmt_n(JitBackend::LLVM, VarType::Void,
-                               occluded ? "// Ray trace (occluded)"
-                                        : "// Ray trace",
+                               within_vcall ? (occluded ? "// Ray trace (occluded) (within_vcall)"
```

Comment:
can remove the ``_`` in the comment.

Reply:
Since you are making the change, would you mind renaming `occluded` to `shadow ray` (and `shadow_ray` in the parameter)? Occluded sounds weird to me, because it sounds like an occlusion has already been found.